setup.py

```diff
@@ -61,7 +61,7 @@
 builtins._ASTROPY_PACKAGE_NAME_ = PACKAGENAME
 
 # VERSION should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
-VERSION = '0.5.0.dev'
+VERSION = '0.6.0.rc1'
```

Comment:
Suggestion: change to `0.6.0rc1` for consistency (e.g. with https://www.python.org/dev/peps/pep-0440/#public-version-identifiers)